Sources/SwiftDocC/Model/Rendering/RenderContentCompiler.swift

```diff
@@ -77,6 +78,8 @@ struct RenderContentCompiler: MarkupVisitor {
         let imageIdentifier: RenderReferenceIdentifier = .init(unescapedSource)
         if let resolvedImages = context.resolveAsset(named: unescapedSource, in: identifier) {
             imageReferences[unescapedSource] = ImageReference(identifier: imageIdentifier, altText: image.altText, imageAsset: resolvedImages)
+        } else if unescapedSource.contains("://"){ // FIXME: - Maybe there is a better way to check scheme
```

Comment:
Is there some way that the context could be responsible for resolving the external reference? 

For example, maybe the context could return an empty asset value for the external asset or the context could have another method that creates and returns the reference.

Would this work if the check is the reference is an external image were moved into `context.resolveAsset(named: unescapedSource, in: identifier)` instead?